common/http/Cargo.toml

```diff
@@ -9,23 +9,31 @@ edition = "2018"
 metrics = { package = "metrics", path = "../metrics" }
 state = { package = "state", path = "../state" }
 #http
-logdna-client = { git = "https://github.com/logdna/logdna-rust.git", branch="0.5.x", version = "0.5" }
+logdna-client = { git = "https://github.com/logdna/logdna-rust.git", branch="len", version = "0.5" }
```

Comment:
Friendly reminder to avoid forgetting to change it once merged :)

Reply:
Yeh, need to PR that before we can merge these. Thanks!